ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1689,3 +1689,12 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+FCU_DFMSC_DSP_ZTC:
```

Comment:
`DFSMC` should have `DFSS` with it. Shouldn't need `ZTC` if you have `DSP`. Can you instead use `FCU_DFSS_DFMSC_DSP_DXZC` or `FCU_DFSS_DFMSC_DSP_HPZC`?

Reply:
This is an incomplete canonical type because you have not identified what component is used to control the temperature (for example: `DXZC` or `HPZC`)